errors/index.js

```diff
@@ -0,0 +1,17 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err.status && err.msg) {
+      res.status(err.status).send({ msg: err.msg });
+    } else next(err);
+  };
+  
+  exports.handlePsqlErrors = (err, req, res, next) => {
+    if (err.code === '22P02') {
+      res.status(400).send({ msg: 'Invalid input' });
+    }
+    else next(err);
+  };
```

Comment:
As  per my other comment this is unneeded for this endpoint and hasn't  been TDD'd, please remove and check for other similar code.